lib/widget/building_popup.dart

```diff
@@ -0,0 +1,279 @@
+import 'package:flutter/material.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:sliding_up_panel/sliding_up_panel.dart';
+import 'package:concordi_around/provider/map_notifier.dart';
+import 'package:provider/provider.dart';
+import 'package:concordi_around/service/map_constant.dart' as constant;
+import 'package:concordi_around/model/coordinate.dart';
+import 'package:concordi_around/widget/search/main_search_bar.dart';
+
+class BuildingPopup extends StatefulWidget {
+  State<StatefulWidget> createState() {
+    return _BuildingPopupState();
+  }
+}
+
+class _BuildingPopupState extends State<BuildingPopup> {
+  List<String> _pictures = List<String>();
+  LatLng currentPlace =  LatLng(0, 0);
+  String _name = '';
+  String _address = '';
+  String _phone = '';
+  String _website = '';
+  String _openClosed = '';
+
+  @override
+  Widget build(BuildContext context) {
+    MapNotifier mapNotifier = Provider.of<MapNotifier>(context);
+
+    if(currentPlace != mapNotifier.selectedLatlng){
+      currentPlace = mapNotifier.selectedLatlng;
+      _getBuildingDetails(currentPlace);
+    }
+    
+    BorderRadiusGeometry radius = BorderRadius.only(
+      topLeft: Radius.circular(24.0),
+      topRight: Radius.circular(24.0),
+    );
+  
+    return Visibility(
+      visible: mapNotifier.showInfo,
+      child: SlidingUpPanel(
+        minHeight: 80,
+        maxHeight: 450,
+        borderRadius: radius,
+        backdropEnabled: true,
+        panelBuilder: (ScrollController sc) => _scrollingList(sc),
+        collapsed: Container(
+          padding: EdgeInsets.fromLTRB(100, 0, 0, 0),
+          child: Row(
+            mainAxisSize: MainAxisSize.max,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: <Widget>[
+              IconButton(
+                icon: Icon(Icons.close),
+                onPressed: () => {
+                  mapNotifier.setPopupInfoVisibility(false)
+                },
+              ),
+            ],
+          )
+        ),
+      )
+    );
+  }
+
+  Widget _scrollingList(ScrollController sc){
+    return Stack(
+      children: <Widget>[
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            Icon(
+              Icons.maximize,
+              color: Colors.grey[350],
+              size: 40,
```

Comment:
Everything looks good, but if you can remove these two lines it will make it consistent with the direction panel. Thanks